apps/condo/domains/property/gql.js

```diff
@@ -19,6 +19,16 @@ const PROPERTY_FIELDS = `{ name address addressMeta { ${ADDRESS_META_SUBFIELDS_Q
 const PROPERTY_TABLE_FIELDS = `{ ${COMMON_FIELDS} unitsCount uninhabitedUnitsCount addressMeta { ${ADDRESS_META_SUBFIELDS_TABLE_LIST} }  ticketsInWork }`
 const Property = generateGqlQueries('Property', PROPERTY_FIELDS)
 const PropertyTable = generateGqlQueries('Property', PROPERTY_TABLE_FIELDS)
+const PROPERTY_ID_ONLY_FIELDS = '{ id }'
+const PropertyIdOnly = generateGqlQueries('Property', PROPERTY_ID_ONLY_FIELDS)
+const PROPERTY_ORGANIZATION_ID_FIELDS = '{ id organization { id } }'
+const PropertyOrganizationId = generateGqlQueries('Property', PROPERTY_ORGANIZATION_ID_FIELDS)
+const PROPERTY_ADDRESS_AND_ADDRESS_KEY_FIELDS = '{ id address addressKey }'
+const PropertyAddressAndAddressKey = generateGqlQueries('Property', PROPERTY_ADDRESS_AND_ADDRESS_KEY_FIELDS)
+const PROPERTY_ADDRESS_AND_MAP = `{ id address map { ${PROPERTY_MAP_JSON_FIELDS} } }`
+const PropertyAddressAndMap = generateGqlQueries('Property', PROPERTY_ADDRESS_AND_MAP)
+const PROPERTY_FOR_DISCOVER_SERVICE_CONSUMER = '{ organization { id } address }'
+const PropertyForDiscoverServiceConsumersService = generateGqlQueries('Property', PROPERTY_FOR_DISCOVER_SERVICE_CONSUMER)
```

Comment:
Should we always specify the `only` postfix for such truncated queries?
`PropertyIdOnly`,
`PropertyOrganizationIdOnly`,
`PropertyAddressAndAddressKeyOnly`,
...

Reply:
There was a discussion recently and we came to this pattern,`<UsageMutationName><ModelName>` or `<ModelName><FieldName>Only`